### PR TITLE
ensure prow pushes latest tag to gcr.io/lololololol-eggplant per stev…

### PR DESCRIPTION
**Pull request by BenTheElder**
…e's request

/assign @ixdy @fejta 

FYI @stevekuznetsov 

> if those images get tagged with the eggplant emoji and put into gcr.io/lololololol i'm fine with that as long as I can rely on the latest being there" 

:heavy_check_mark: 

also buildifier-ed the prow defs

### CODE REVIEW

**Comment by ixdy** [comment]
/kind :hankey: 

seems like this might break autobump, which presumably expects the images to still go to gcr.io/k8s-prow?

**Comment by krzyzacy** [comment]
I would say I'm ok if we mirror the image to a second gcr but not change the default one? We/other people have other prow deployments relies on those images...

**Comment by BenTheElder** [comment]
/retest

I don't think it should break autobump


Those images still work, we're not deleting anything. This is a dedicated registry / project

**Comment by ixdy** [comment]
This will change the default from `gcr.io/k8s-prow` to `gcr.io/lolol[I don't even know how many times to do this]lol-eggplant`. I don't think we should do this.

**Comment by BenTheElder** [comment]
/shrug

it should have worked fine, the default is only for automation anyhow. it doesn't matter that it's `"l" + ("ol" * 5)` :^)

**Comment by stevekuznetsov** [comment]
oh boy

File: hack/print-workspace-status.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ docker_tag="v${build_date}-${git_commit}"
 # TODO(fejta): retire STABLE_PROW_REPO
 cat <<EOF
 STABLE_DOCKER_REPO ${DOCKER_REPO_OVERRIDE:-gcr.io/k8s-testimages}
-STABLE_PROW_REPO ${PROW_REPO_OVERRIDE:-gcr.io/k8s-prow}
+STABLE_PROW_REPO ${PROW_REPO_OVERRIDE:-gcr.io/lololololol-eggplant}
```
Comment (fejta) [inline hack/print-workspace-status.sh]: Please revert this change (and the prow/def.bzl change)

**Comment by fejta** [comment]
This horrible PR needs more thumbs down.

**Comment by BenTheElder** [comment]
This is not a very real PR :^)

It turns out we should be pushing latest images though, so consider this solved.
